lldb/test/Shell/SymbolFile/DWARF/x86/dwp-foreign-type-units.cpp

```diff
@@ -78,8 +78,8 @@
 // DWPFOO-NEXT: struct CustomType {
 // DWPFOO-NEXT:     typedef unsigned int IntegerType;
 // DWPFOO-NEXT:     typedef float FloatType;
-// DWPFOO-NEXT:     CustomType::IntegerType x;
-// DWPFOO-NEXT:     CustomType::FloatType y;
+// DWPFOO-NEXT:     IntegerType x;
+// DWPFOO-NEXT:     FloatType y;
```

Comment:
Seems like a good improvement

Reply:
Again, this was never what was written in source code. It was fully qualified instead. Now it's always unqualified, which coincidentally matches what was written.